### ISSUE TITLE
Playback view slows down to a crawl

### ISSUE THREAD

**Issue by serranojavier** [ekky-llc/spt-raid-review#44]
Hi loving the mod for seeing my deaths,

I'm having an issue though, I upgraded to v .06 deleted the correct .dll plugin and mod folder and migrated my data folder to the new usr/mods folder.

The issue however is that when I open a raid for playback on my chrome browser it starts out playing fine at normal speed but then it gets really laggy as if it was playing back at 10fps or lower. 

What logs might you need to see?

**Comment by christopher-talke** [comment]
I won't need to see any logs, I have been working on improving this.

If you check the releases on the main page of this repository, there is a 0.1.0_test release.

Did you want to give that a try see if there is a noticeable performance difference?

**Comment by serranojavier** [comment]
Gave it the test release a try and I'm even though I migrated my data it shows my profile and I can see the saved raids, however upon clicking on any raid all I see is an empty map, no name list and no events logged on the timeline, pressing play doesn't do anything

**Comment by KonstAl54** [comment]
> Gave it the test release a try and I'm even though I migrated my data it shows my profile and I can see the saved raids, however upon clicking on any raid all I see is an empty map, no name list and no events logged on the timeline, pressing play doesn't do anything

same

**Comment by christopher-talke** [comment]
> Gave it the test release a try and I'm even though I migrated my data it shows my profile and I can see the saved raids, however upon clicking on any raid all I see is an empty map, no name list and no events logged on the timeline, pressing play doesn't do anything

Okay I see - one sec I'll upload a new version with that fixed.

**Comment by RebeliusGaming** [comment]
Hi. As mine was really slow too, I tried this, and with the old raids, I have no info and get this error
![image](https://github.com/ekky-llc/spt-raid-review/assets/94674848/d1192d37-a6d4-4a9d-9ec2-da24a59ebbd5)

I'ma play some raids now, and Ill post if it works.

I dont mind losing old data, should I delete the data folder inside mods/raidreview0.1.0?

thank you :)

**Comment by christopher-talke** [comment]
@RebeliusGaming @serranojavier @KonstAl54 

Please try this, the previous version I uploaded I forgot to re-add some critical code to the Playback.

https://github.com/ekky-llc/spt-raid-review/releases/tag/v0.1.0_test_a

Sincere apologies for screwing you guys around.

**Comment by RebeliusGaming** [comment]
Should I just extract it and overwrite if asked?

**Comment by christopher-talke** [comment]
It shouldn't ask for an overwrite, it should make a new folder in your `user/mods/` and you need to copy your /data folder from the old one as per install notes.

Might just be because it's a very fresh upload, however, if you don't trust, I've included a scan result in the download page.

Otherwise not sure why it's complaining 😞

**Comment by RebeliusGaming** [comment]
Works FLAWLESS right now.

I dont know if you get access to this data, but also being able to watch bullets would be great!

**Comment by RebeliusGaming** [comment]
Sorry
Spoke too early. Same problem again. Nice at start, but really slow after some seconds

**Comment by christopher-talke** [comment]
> Sorry
> Spoke too early. Same problem again. Nice at start, but really slow after some seconds

Yeah, I figured that would happen, I'm overloading the browser too hard.

I tried switching the rendering approach, which broke the last build, and I need to do some more research around a better way.

I'll keep this Issue open for now.

**Comment by serranojavier** [comment]
Just tried v0.1.1 and performance is great now! Love this mod to see how I've died

**Comment by ekky1328** [comment]
> Just tried v0.1.1 on 3.8.3 and performance is great now! Love this mod to see how I've died

That's great news, glad you like it man :)
